prow/prowjobs/GoogleCloudPlatform/gcp-guest/gcp-guest-config.yaml

```diff
@@ -554,7 +554,7 @@ periodics:
d/global/images/family/debian-12,projects/debian-cloud/global/images/family/debian-11-arm64,projects/debian-cloud/global/images/family/debian-12-arm64,projects/centos-cloud/global/images/family/centos-7,projects/centos-cloud/global/images/family/centos-stream-8,projects/centos-cloud/global/images/family/centos-stream-9,projects/rhel-cloud/global/images/family/rhel-7,projects/rhel-cloud/global/images/family/rhel-8,projects/rhel-cloud/global/images/family/rhel-9,projects/rhel-cloud/global/images/family/rhel-9-arm64,projects/rocky-linux-cloud/global/images/family/rocky-linux-8,projects/rocky-linux-cloud/global/images/family/rocky-linux-8-optimized-gcp,projects/rocky-linux-cloud/global/images/family/rocky-linux-8-optimized-gcp-arm64,projects/rocky-linux-cloud/global/images/family/rocky-linux-9-arm64,projects/rocky-linux-cloud/global/images/family/rocky-linux-9-optimized-gcp-arm64,projects/almalinux-cloud/global/images/family/almalinux-8,projects/opensuse-cloud/global/images/family/opensuse-leap,projects/suse-cloud/global/images/family/sles-12,projects/suse-cloud/global/images/family/sles-15,projects/suse-cloud/global/images/family/sles-15-arm64,projects/opensuse-cloud/global/images/family/opensuse-leap-arm64,projects/opensuse-cloud/global/images/family/opensuse-leap-arm64,projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts,projects/ubuntu-os-cloud/global/images/family/ubuntu-2204-lts,projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2004-lts,projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2204-lts,projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2304-amd64,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1604-lts,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1804-lts,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-2004-lts,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-2204-lts,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-fips-1804-lts,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-fips-2004-lts,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-1804-lts-arm64,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-2004-lts-arm64,projects/ubuntu-os-pro-cloud/global/images/family/ubuntu-pro-2204-lts-arm64,projects/ubuntu-os-cloud/global/images/family/ubuntu-2004-lts-arm64,projects/ubuntu-os-cloud/global/images/family/ubuntu-2204-lts-arm64,projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2004-lts-arm64,projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2204-lts-arm64,projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2304-arm64,projects/windows-cloud/global/images/family/windows-2016,projects/windows-cloud/global/images/family/windows-2016-core,projects/windows-cloud/global/images/family/windows-2019,projects/windows-cloud/global/images/family/windows-2019-core,projects/windows-cloud/global/images/family/windows-2022,projects/windows-cloud/global/images/family/windows-2022-core"
```

Comment:
cloud/global/images/family/ubuntu-minimal-2304-amd64 is still in the list. Thank you for giving me something to find.

Reply:
and projects/ubuntu-os-cloud/global/images/family/ubuntu-minimal-2304-arm64